documentation/working-with-testcafe-studio/recording-tests/test-actions/on-page-actions/README.md

```diff
@@ -22,42 +22,42 @@ The **On-Page Actions** category contains test actions that simulate user activi
 
 You can add these actions in two ways:
 
-* Add them to a test and specify its parameters manually.
+* Add them to a test and specify their parameters manually.
 * [Record them](../../README.md). You can record most of the on-page actions by interacting with a tested webpage. TestCafe Studio automatically adds corresponding on-page actions with the necessary parameters to a test.
 
 ## Click
 
-Clicks a webpage element with the left mouse button.
+Click a webpage element (left mouse button).
```

Comment:
1. Why we can't say "clicks"?  This means: "The action clicks a webpage element", we just skip "The action". In addition, other topics about actions (assertions, browser actions, debug) contain _verb+"s"_ in the actions' description.
2. 
>Click a webpage element (left mouse button).

Why we need the parentheses? This is the whole description of the method. This phrase "Clicks * with the left mouse button." is commonly used.  For example, https://msdn.microsoft.com/en-us/library/windows/desktop/bb760490(v=vs.85).aspx.

Reply:
Because "You can add these actions in two ways:" You click, not 'clicks'. In this context, it's not the 'action' that does something, the something you do is the action. In any way, to say 'the action clicks/does something' is like saying the action performs an action.

Why parentheses? Because 'click with the left mouse button'; 'click with the right mouse button'-> 'Click this (this button)';'Click that (that button)'.

Reply:
No. The context is "The ... category contains test actions that simulate user activity on a tested web page." Then, you follow the links to see the action description. This is a reference topic. 

And "Clicks a webpage element with the left mouse button" is what the test action (test command) does when you add it to the test. I think we shouldn't repeat "The action clicks ...." in each section. 

As for parentheses, "Clicks ... with the left mouse button" is commonly used phrase in MSDN. I think we should use it.

Reply:
The left mouse button clicks a webpage element

Reply:
No, TestCafe Studio emulates a click on a webpage element with the left mouse button. Why we can't use the commonly used phrase?

Reply:
Ok, use the original